lls/main.c

```diff
@@ -200,13 +206,14 @@ put_nd(struct in6_addr *ip_be, unsigned int lcore_id)
 }
 
 int
-submit_nd(struct rte_mbuf *pkt, struct gatekeeper_if *iface)
+submit_nd(struct rte_mbuf **pkts, int num_pkts, struct gatekeeper_if *iface)
 {
 	if (nd_enabled(&lls_conf)) {
```

Comment:
Yes, I think you're right that it's unnecessary -- it's just un-caught artifact of the previous way of handling ND packets. Thanks!